src/osx/dataview_osx.cpp

```diff
@@ -770,11 +770,18 @@ void wxDataViewCtrl::OnMouse(wxMouseEvent& event)
     event.Skip();
 }
 
+void wxDataViewCtrl::OnChar(wxKeyEvent& event)
+{
+    // generic key handling can moved here
+    event.Skip();
+}
+
```

Comment:
Yes, I wanted to have this as a placeholder, in case you know of something it should support but doesn't yet, this would be 'neutral' place to add event handling